app/src/main/java/com/voipgrid/vialer/logging/RemoteUncaughtExceptionHandler.java

```diff
@@ -17,6 +17,10 @@
         mDefaultHandler = Thread.getDefaultUncaughtExceptionHandler();
     }
 
+    public RemoteUncaughtExceptionHandler(RemoteLoggingActivity remoteLoggingActivity) {
```

Comment:
This isn't needed to just remove this constructor and update so "this" isn't being passed to it when the constructor is being called